specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/AzureMachineLearning.json

```diff
@@ -0,0 +1,12930 @@
+        "imageResponseBaseTypeDiscriminator": {
+          "description": "Property used by Swagger for polymorphic object generation.",
+          "type": "string"
+        },
+        "imageType": {
+          "description": "The type of the image.",
+          "enum": [
+            "Docker",
+            "FPGA"
+          ],
+          "example": "Docker",
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ImageType",
+            "oldModelAsString": true
+          }
+        },
+        "kvTags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The image tag dictionary. Tags are mutable.",
+          "type": "object"
+        },
+        "modelDetails": {
+          "description": "The list of models.",
+          "items": {
+            "$ref": "#/definitions/Model"
+          },
+          "type": "array",
+          "uniqueItems": false
+        },
+        "modelIds": {
+          "description": "The list of model Ids.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array",
+          "uniqueItems": false
+        },
+        "modifiedTime": {
+          "description": "The time the image was last modified.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "name": {
+          "description": "The image name.",
+          "example": "myimage:3",
+          "type": "string"
+        },
+        "properties": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The image properties dictionary. Properties are immutable.",
+          "type": "object"
+        },
+        "tags": {
+          "description": "The image tag list.",
+          "example": "[\"docker\"]",
+          "items": {
+            "type": "string"
+          },
+          "type": "array",
+          "uniqueItems": false
+        },
+        "version": {
+          "description": "The image version.",
+          "example": "3",
+          "format": "int64",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "imageResponseBaseTypeDiscriminator"
+      ],
+      "type": "object"
+    },
+    "InnerErrorResponse": {
+      "description": "A nested structure of errors.",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string"
+        },
+        "innerError": {
+          "$ref": "#/definitions/InnerErrorResponse",
+          "description": "A nested list of inner errors. When evaluating errors, clients MUST traverse through all of the nested “innererrors” and choose the deepest one that they understand."
+        }
+      },
+      "type": "object"
+    },
+    "JsonPatchOperation": {
+      "properties": {
+        "from": {
+          "type": "string"
+        },
+        "op": {
+          "type": "string"
+        },
+        "path
```

Comment:
```suggestion
    "/artifact/v2.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/artifacts/batch/copy/fileSas/{origin}/{container}": {
```

Reply:
our current style is all lowercase.  Leaving as is for now.